packages/styles/scss/components/card-link/index.scss

```diff
@@ -17,12 +17,21 @@
     text-decoration: none;
     display: flex;
     flex-direction: column;
-    padding: $spacing-05;
 
     > *:not(:last-child) {
       margin-bottom: $spacing-07;
     }
 
+    &__wrapper {
+      padding: $spacing-05;
+    }
+
+    &__image {
+      width: 100%;
+      height: auto;
+      object-fit: cover;
```

Comment:
images look a little small, I thought you had a height set to them? 240px? and looks like there's supposed to be a border between the cards

<img width="1078" alt="Screen Shot 2020-01-08 at 10 03 02 AM" src="https://user-images.githubusercontent.com/54281166/71988605-32165500-31fe-11ea-8ad5-8a6b11e49b33.png">

Reply:
yeah @annawen1 as per @photodow I have removed the gray border on the cards. And I have changed the height to `auto` instead of having a specific height.

Reply:
I thought @photodow was referring to use of `outline`? Not sure if we wanted to remove the spacing/border from the designs though. @photodow?

Reply:
I'm fine with going ahead and merging it now since this PR has been open for a while. We can open a ticket to address the border.

Reply:
I'm fine with going ahead and merging it now since this PR has been open for a while. We can open a ticket to address the border.

Reply:
@mkothur naw, @annawen1's right. We need the spacing inbetween. That comes from the Condensed Grid. We don't have to build that into each component, we just have to use what's already there. If Carbon decides one day they want to change the condensed grid to 3px, then we have to go to a bunch of components and patterns to make sure we updated.

Here are a couple resources...
https://carbon-elements.netlify.com/grid/examples/preview/#condensed-grid
https://www.carbondesignsystem.com/guidelines/2x-grid/implementation/#the-condensed-grid

Reach out if you have any questions or concerns.

Reply:
As for the height, we're going to need a specific ratio for the images if I'm not mistaken.

Reply:
New issue created to track this: https://github.com/carbon-design-system/ibm-dotcom-library/issues/976